### DISCUSSION TITLE
Simple improvement: Show percentage & celcius symbols on Tray Icons

### DISCUSSION THREAD

**Discussion by PennyWafel** [Rem0o/FanControl.Releases#2666] [Ideas]
Hi, great piece of software!

 I would like to request a simple UI improvement for the Tray Icons. The tray icons currently show only the numbers, but it is not clear whether that number represents a fan speed percentage or a temperature in celcius. For users such as myself, who want to see both the temps as well as the fan speed, it can be difficult to tell which is which.

It would be really great if the tray icons would simply show °C if it's a temperature readout and a % if it's a fan speed readout. 

Thanks for making this software.

**Comment by Rem0o** [comment]
They do if you hover your mouse over:
![image](https://github.com/user-attachments/assets/1d4e8d9a-fb96-4f3d-bd6d-81cdb77d97ac)

Otherwise there is no room to put the unit within the tray icon itself.

**Reply by PennyWafel** [reply]
Thanks for the reply and explanation. As an alternative, would it perhaps be possible to add more font options? Such as choosing which font to display in the tray or making it bold, underlined or italic? That would allow users to add more differentiating features to tray icons for easier identifying.

**Reply by Rem0o** [reply]
Most if not all fonts do not work well in a 16x16 square.  Positioning would also be required making the whole thing really hard to configure. I prefer not going into that direction.
